templates/devel/profile.html

```diff
@@ -23,5 +23,20 @@ <h2>Developer Profile</h2>
         <p><label></label> <input title="Save changes" type="submit" value="Save" /></p>
     </form>
 
+    <form id="api-profile-form" enctype="multipart/form-data" method="post" action="">{% csrf_token %}
+        <h3>API token</h3>
+        <p>Token for completing todolist items with for example, rebuild-todo</p>
+        {% if profile.api_token is None %}
```

Comment:
I assume it would be a good pattern to be able to revoke a token, or to phrase it differently: Generate a new token and invalidate the existing one.
I'd probably propose something like only showing the token once when the generate button is clicked and otherwise show an invalidate/regenerate button instead. We should hide the token after generating it.